src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  padding-left: 50px;
```

Comment:
remove it, you need to add paddings for both sides to the header (left and right)